kustomize.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cat <&0 > /home/shell/helm-run/all.yaml
```

Comment:
Whats point of this ?

Reply:
It creates an all.yaml which is an aggregation of all the manifests, this is also done here: https://github.com/rancher/rancher/blob/release/v2.6/package/kustomize.sh#L3. This is what is modified by kustomize before being applied.